app/services/manual/withdraw_service.rb

```diff
@@ -40,8 +40,6 @@ def remove_from_search_index
 
   def manual
     @manual ||= Manual.find(manual_id, user)
-  rescue KeyError => error
-    raise ManualNotFoundError.new(error)
```

Comment:
I've removed the commit that removed this `rescue` statement but I'm not planning to do anything more about it. I'm happy to leave it until we revisit this code.